api/services/NorthstarService.js

```diff
@@ -40,4 +40,8 @@ module.exports = {
     return { type, id };
   },
 
+  isOneOfFieldSet(submission) {
+    return ['user_id', 'email', 'mobile'].every(userField => !submission[userField]);
+  },
+
```

Comment:
I don't think `NorthstarService` is a good place for this function.
This attributes are not related to Northstar, they correspond to incoming POST json data. I'd create a new service and name it something like `ValidationHelperService` or `HelpersService`.